REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,70 @@
+### Remaining public aliases
+
+annotation_reader
+cache.app_clearer
+$commandId
+debug.templating.engine.php
+profiler.request_matcher
+router
+security.access.decision_manager
+security.acl.cache
+security.acl.dbal.connection
+security.acl.provider
+security.encoder_factory
+security.firewall
+security.password_encoder
+session.handler
+session.storage
+session.storage.filesystem
```

Comment:
flagged as "for BC" https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml#L72..L73
should we deprecate it on 3.4? (can't find the PR)